test cases/common/20 global arg/meson.build

```diff
@@ -8,6 +8,7 @@ build_c_args = ['-DARG_BUILD']
 c_args = ['-DARG_HOST']
 
 add_global_arguments('-DMYCANDCPPTHING', language: ['c', 'cpp'])
+add_global_arguments('-Db_lto=true', language: ['c', 'cpp'])
```

Comment:
What is this testing? Unless I'm missing something big, this is passing the `-Db_lto=true` option to the C and C++ compilers. I don't see how this has anything to do with the other patch in this PR.

Reply:
It is exactly the target of this fix, check b_lto option do not return -> 

`ERROR: Unknown options: "b_lto"`

Reply:
Ok I remove it.
My mistake.

These test I wanted to add are already cover by thoses inside

https://github.com/mesonbuild/meson/blob/8967090149cb89f7b82fc6c8f72764975912fa58/unittests/allplatformstests.py#L1356

Like I said, I don't know exactly how to reproduce the problem.
Does the problem occur with option "-Db_lto=true".
It does on my machine frequently.

Does this patch fix it.
It does by not testing this flag is unknown......